vms/platformvm/validators/manager.go

```diff
@@ -203,6 +209,9 @@ func (m *manager) makePrimaryNetworkValidatorSet(
 	currentValidators, ok := m.cfg.Validators.Get(constants.PrimaryNetworkID)
 	if !ok {
 		// This should never happen
+		m.log.Error(ErrMissingValidator.Error(),
+			zap.Stringer("networkID", constants.PrimaryNetworkID),
```

Comment:
`networkID` is a `uint32` - this is a `subnetID`
```suggestion
			zap.Stringer("subnetID", constants.PrimaryNetworkID),
```